webserver/Program.cs

```diff
@@ -55,11 +59,26 @@
 app.UseHttpsRedirection();
 app.UseStaticFiles();
 
+app.MapControllers();
+
 app.UseRouting();
 
 app.UseAuthentication();
 app.UseAuthorization();
 
 app.MapRazorPages();
 
+app.MapControllerRoute(
+    name: "default",
+    pattern:"{controller=Home}/{action=Index}/{id?}"
+);
```

Comment:
You are not adding the Home controller in this PR, does it already exist?
If not, I don't think this code should be added.

Reply:
Doesn't exist. Code wasn't needed, i tested. Removed